src/recipes/vscode/settings.mjs

```diff
@@ -42,7 +42,7 @@ export const getSettings = async (settingsPath) => {
 
     return {
       fileExists: true,
-      settings: JSON.parse(file),
+      settings: parse(file.toString()),
```

Comment:
Rather than doing a `toString()` here, I think you can do `await readFile(settingsPath, "utf8")` above?

Reply:
Yeah. That seems better.